component/gitlab-ci.jsonnet

```diff
@@ -0,0 +1,83 @@
+local kap = import 'lib/kapitan.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.openshift4_cloudscale;
+
+{
+  'gitlab-ci': {
+    stages: [
+      'validate',
+      'plan',
+      'deploy',
+    ],
+    default: {
+      image: params.images.terraform.image + ':' + params.images.terraform.tag,
+      tags: ['unpriv_dockerbuild'],
```

Comment:
This only makes sense for the git.vshn.net GitLab instance, job tags should probably be configurable.